database/sql/src/main/scala/cromwell/database/slick/MetadataSlickDatabase.scala

```diff
@@ -201,10 +201,11 @@ class MetadataSlickDatabase(originalDatabaseConfig: Config)
                                       startTimestampOption: Option[Timestamp],
                                       endTimestampOption: Option[Timestamp],
                                       page: Option[Int],
-                                      pageSize: Option[Int])
+                                      pageSize: Option[Int],
+                                      submissionTimestampOption: Option[Timestamp])
                                      (implicit ec: ExecutionContext): Future[Seq[WorkflowMetadataSummaryEntry]] = {
     val action = dataAccess.queryWorkflowMetadataSummaryEntries(workflowStatuses, workflowNames, workflowExecutionUuids,
-      labelAndKeyLabelValues, labelOrKeyLabelValues, startTimestampOption, endTimestampOption, page, pageSize).result
+      labelAndKeyLabelValues, labelOrKeyLabelValues, startTimestampOption, endTimestampOption, page, pageSize, submissionTimestampOption).result
```

Comment:
TOL nitpick but in parameter lists that include submission, start, and end timestamps I'd lean toward putting all three next to each other in that (chronological) order.